support/k8s/k8s-workload-registrar/controller.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	admv1 "k8s.io/api/admission/v1"
```

Comment:
This belongs in the spot where the old admv1beta1 was.

Reply:
thank you, it should be fixed now.